src/settings/SettingsStore.ts

```diff
@@ -576,7 +610,7 @@ export default class SettingsStore {
         return handlers;
     }
 
-    static _getFeatureState(settingName) {
+    private static getFeatureState(settingName: string): "labs" | "disable" | "enable" | string {
```

Comment:
can we get an Enum for this?

Reply:
I'll declare it as a `type`. It's not going to survive long.